Nodejs/Product/Npm/SPI/NodeModules.cs

```diff
@@ -34,12 +34,15 @@ internal class NodeModules : AbstractNodeModules {
                 foreach (var dependency in parentPackageJson.AllDependencies) {
                     Package module = null;
                     if (!Contains(dependency.Name)) {
-                        module = new Package(
-                            parent,
-                            Path.Combine(modulesBase, dependency.Name),
-                            showMissingDevOptionalSubPackages);
-                        if (parent as IPackage == null || !module.IsMissing || showMissingDevOptionalSubPackages) {
-                            AddModule(module);
+                        var dependencyPath = Path.Combine(modulesBase, dependency.Name);
+                        if (dependencyPath.Length <= 224) {

```

Comment:
Right now I recall why I do that, I probably don't want to dig deeper in the Package put checks in this RootPackage, since I not 100% understand how this works. But if needed i could try.